.github/workflows/build.yml

```diff
@@ -83,29 +83,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        pkg:
+          - ./client ./cmd/buildctl ./worker/containerd ./solver
+          - ./frontend
```

Comment:
looks like maybe even `./frontend` can be together with the rest. Even the sum of the first 2 should be smaller than `./frontend/dockerfile` on its own.